metals/src/main/scala/scala/meta/internal/metals/codeactions/MillifyDependencyCodeAction.scala

```diff
@@ -24,7 +24,7 @@ class MillifyDependencyCodeAction(buffers: Buffers) extends CodeAction {
     val range = params.getRange()
 
     val tokenized =
-      if (path.isScalaScript && range.getStart == range.getEnd)
+      if ((path.isScalaScript || path.isMill) && range.getStart == range.getEnd)
```

Comment:
I don't think this will work currently with Mill, it's for `//> using dep org.something %% name % version` to change it.

We could add that option to Mill build files, but it would require a bit of work (not much I think)